models/doctype/Uom/Uom.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+    name: "Uom",
+    doctype: "DocType",
+    naming: "random",
```

Comment:
Naming of UOM should not be random! Should be the `name` property. See Item or Party